docs/_data/prague-2022-config.yaml

```diff
@@ -67,6 +67,8 @@ buttons:
     #   link_absolute: https://www.youtube.com/playlist?list=PLZAeFn6dfHpkBJAPYFrob6gqdiBuePwGJ
 
 tickets:
+  concierge:
+    price: €350
```

Comment:
This seems a bit steep. I wonder about lowering the price, but having a limit of at least 3 tickets? I'm also fine with pricing higher but not having a limit, but seems more likely that these folks will want multiple tickets.

Reply:
Is it technically possible to have a minimum ticket amount on Tito? If so, then we can probably do 250 with a minimum of 3-4 tickets..

Reply:
These don't go through tito, that's the point ;)

€ 250 with a 3 ticket minimum? Don't want to make it too attractive either, because some corporates can use the normal tito process with more effort on their side, we want to keep that as the ideal option because it scales well.

Reply:
Yea, I was just looking at the Djangocon tickets, and there's are $200 more, but that's only ~25% of the total cost. I think a minimum of 300 EUR for dealing with it feels like plenty, and makes it a bit less overwhelming price-wise?